src/interfaces/packagingSObjects.ts

```diff
@@ -346,4 +346,11 @@ export namespace PackagingSObjects {
     IsDeprecated: boolean;
     MetadataZip?: string;
   };
+
+  export type MetadataPackage = {
```

Comment:
Needed for querying the package name for any dependencies needed by a retrieved package version.